auth.md

```diff
@@ -0,0 +1,41 @@
+# Authentification
+
+Marauder's map uses [JWTs](https://jwt.io/) to authenticate users.
+
+## Usage
+
+
+## Structure
+### Header
+
+```json
+{
+  "alg": "RS512",
```

Comment:
thoughts about this?

Reply:
Sieht gut aus! Das ist dann der Token für die Authentifikation, wenn es einen Anmeldeversuch gibt oder?

Reply:
Passt! Gibt es einen wichtigen Unterschied zwischen RS256 und RS512? Nur, dass die Verschlüsselung in 128-Bit im Falle von RS256 ist und 256-Bit für RS512?